documentation/src/main/asciidoc/topics/con_clients_unmarshalled_objects.adoc

```diff
@@ -0,0 +1,14 @@
+[id='clients-unmarshalled-objects_{context}']
+= Clients and POJOs
+Even though {brandname} does not recommend doing so, clients can operate on caches that store unmarshalled POJOs with the `application/x-java-object` media type.
+
+[discrete]
+== Hot Rod clients
+
+Hot Rod client marshallers must be available to {brandname} Server so it can deserialize your Java objects.
```

Comment:
Actually, even though we wouldn't recommend it, a Hot Rod client could use the same JSON or XML conversion that REST clients use ;)

Reply:
yeah, maybe let's not go there. :sunglasses: